src/components/room/RoomFooter.jsx

```diff
@@ -1,12 +1,15 @@
 import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 
-function RoomFooter({ price }) {
+function RoomFooter({ price, host, title }) {
   return (
     <StyledRoomFooter>
       <div>
         <span>₩{price}</span> / 박
       </div>
-      <button>예약하기</button>
+      <Link to="/" state={{ title: { title }, host: { host } }}>
+        <button>예약하기</button>
+      </Link>
```

Comment:
title, host 따로 전달하지 않고 roomInfo로 전달해도 좋을 것 같습니다
```suggestion
      <Link to="/" state={{ roomInfo: { title, host } }}>
        <button>예약하기</button>
      </Link>
```

Reply:
roomInfo 로 전달하게 되면 Room 컴포넌트 에서도 roomInfo 로 넘겨줘야 할까??? 이대로만 수정하니 정보가 안넘어오네
일단 좀더 고민해볼게!!

Reply:
`Uncaught TypeError: Cannot destructure property 'roomInfo' of 'state' as it is null.` 이게 뜨고 있었네 😂
state가 null일 수도 있는데 구조 분해 할당하려고 하니까 뜨는 타입 에러여서 아래처럼 수정하면 돼!

```javascript
function Home() {
  const location = useLocation();
  const state = location.state;

  return (
    <StyledHome>
      <HomeInfo title={state?.roomInfo.title ?? 'unBooked'} host={state?.roomInfo.host ?? 'unBooked'} />
      <NavBar />
    </StyledHome>
  );
}
```